src/sourcegraph.d.ts

```diff
@@ -785,6 +785,37 @@ declare module 'sourcegraph' {
         ): Unsubscribable
     }
 
+    /**
+     * A query transformer alters a user's search query before executing a search.
+     *
+     * Query transformers allow extensions to define new search query operators and syntax, for example,
+     * by matching strings in a query (e.g. `go.imports:`) and replacing them with a regular expression or string.
+     */
+    export interface QueryTransformer {
+        /**
+         * Transforms a search query into another, valid query. If there are no transformations to be made
```

Comment:
I'd replace "the original query is returned" with ", it should return the original query"